tool_test.go

```diff
@@ -0,0 +1,165 @@
+package codacytool
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+)
+
+func setup() {
+	os.Setenv(_basePathEnvVar, testsResourcesLocation)
+}
+func shutdown() {
+	os.Unsetenv(_basePathEnvVar)
```

Comment:
.... and variables set by a go process won't spill over to the system level, so this is instruction is not actually effective.
Still I would not rely on env vars at all :-)

Reply:
You're right, setting and unsetting environment variables is not a good idea. Removed it ✅